Mathlib/Algebra/DirectSum/Basic.lean

```diff
@@ -150,6 +150,12 @@ theorem sum_support_of [∀ (i : ι) (x : β i), Decidable (x ≠ 0)] (x : ⨁ i
   DFinsupp.sum_single
 #align direct_sum.sum_support_of DirectSum.sum_support_of
 
+theorem sum_univ_of [Fintype ι] (x : ⨁ i, β i) :
+    ∑ i ∈ Finset.univ, of β i (x i) = x := by
+  change (∑ i ∈ Finset.univ, DFinsupp.single i (x i)) = x
```

Comment:
Do we really need this `change`? If so we are maybe missing a lemma.

Reply:
I tried to add various rewrite lemmas, even `lemma foo [Fintype ι] (x : ⨁ i, β i) : /∑ i ∈ Finset.univ, of β i (x i)) = ∑ i ∈ Finset.univ, DFinsupp.single i (x i)`, but it I can't get it to work without the `change`.